.github/workflows/tenzir.yaml

```diff
@@ -103,7 +103,7 @@ jobs:
           # limit_version. If the version does not exist yet (e.g., because of
           # an intentional break), only check latest. We usually set this to the
           # minimum supported version.
-          limit_version=v4.18.0
+          limit_version=v4.26.0
```

Comment:
This still needs a force merge because "latest" is always taken (even though "v4.26.0" will come after what is "latest" right now).

Reply:
This will cause all tests on main to fail as well, so I think we should consider temporarily disabling latest as well.

Reply:
Should I disable the "latest" job if the version does not exist? It's probably a bit too much, as we'd want to test against latest again once the image from this PR is latest.

Reply:
I think that's a good idea. Shouldn't be hard to set that up.